perf/rt_tests.py

```diff
@@ -52,7 +52,7 @@ def setUp(self):
                 self.cancel('%s is needed for the test to be run' % package)
         tarball = self.fetch_asset(
             "https://www.kernel.org/pub/linux/utils/rt-tests/"
-            "rt-tests-1.0.tar.gz")
+            "rt-tests-1.10.tar.gz")
```

Comment:
Any specific reason for the change or is it just an update? If it is an update, you can make url configurable from yaml file.